### ISSUE TITLE
Add py and feature file in installation docs

### ISSUE THREAD

**Issue by sih4sing5hong5** [behave/behave-django#6]
In <https://pythonhosted.org/behave-django/installation.html>, there is no exactly examples for `your_steps.py`, `environment.py` and `your-feature.feature`.

I think it should be like <http://pythonhosted.org/behave/tutorial.html>, providing `tutorial.py` and `tutorial.feature` at least

**Comment by bittner** [comment]
In fact, there is no difference. You do not need any additional setup apart from adding `behave_django` to your `INSTALLED_APPS` (see the [Installation](https://pythonhosted.org/behave-django/installation.html) chapter). The example for features, steps, environment would be just what the behave docs show. Plus optional goodies from the [Usage](https://pythonhosted.org/behave-django/usage.html) chapter, naturally.

If you feel there is a full-blown example missing, please fork the repo, adapt the docs, and place a pull request.

We could in theory also maintain a list of good examples, open source projects that use (and do a good job using) `behave-django`.

**Comment by bittner** [comment]
The files mentioned (and used for the demo output) in the Installation chapter are actually the ones in the [`features/`](https://github.com/behave/behave-django/tree/master/features) folder of our repository:

* [features/environment.py](https://github.com/behave/behave-django/blob/master/features/environment.py)
* [features/running-tests.feature](https://github.com/behave/behave-django/blob/master/features/running-tests.feature)
* [features/steps/running_tests.py](https://github.com/behave/behave-django/blob/master/features/steps/running_tests.py)

All the files in this folder serve as useful examples (see [context-urlhelper.py](https://github.com/behave/behave-django/blob/master/features/steps/context-urlhelper.py), for example). Maybe we should mention that and link to them from the docs?

**Comment by sih4sing5hong5** [comment]
Adding links for the above three files is enough.

`All the files in this folder serve as useful examples (see context-urlhelper.py, for example).`
The docs can mention and link this.
More examples is better.

**Comment by bittner** [comment]
I think we will link (back) to the [Django chapter](http://python-behave.readthedocs.org/en/latest/django.html) of the `behave` docs, once [PR 429](https://github.com/behave/behave/pull/429) is merged. This will make using `behave-django` much clearer. Without adding explanations that are already provided by the `behave` docs. And in addition we can link to our source code from the docs, as discussed above.

>  More examples is better.

What do you think about adding Django examples to the [behave/behave.example](https://github.com/behave/behave.example/tree/master/features) repository? Please, do that! Everybody will appreciate your contribution.

**Comment by bittner** [comment]
@mixxorz Can you re-generate the docs once the PR on behave is merged and their docs published?

**Comment by mixxorz** [comment]
@bittner Sure thing

**Comment by mixxorz** [comment]
I've rebuilt and uploaded the docs!

https://pythonhosted.org/behave-django/

I'll open up another issue regarding migrating to Read The Docs (for automated doc builds).
